packages/office-ui-fabric-react/src/components/ActivityItem/ActivityItem.tsx

```diff
@@ -65,7 +65,7 @@ export class ActivityItem extends BaseComponent<IActivityItemProps, {}> {
   private _onRenderComments = (props: IActivityItemProps): JSX.Element | null => {
     const classNames = this._getClassNames(props);
 
-    const comments = props.comments || props.commentText;
+    const comments = props.comments;
```

Comment:
Hmm, I don't think we want to remove our own deprecated usage.. this will break backwards compat until we can actually remove these items. I think we need a way to exclude our own deprecations.

Reply:
I see, I think I did that in about 3 or 4 places. I can go and change them back to how they were before. Should be straightforward.

Reply:
Just pushed a commit that reverts this type of changes back to how they were before.